Octokit.Reactive/Clients/IObservableWatchedClient.cs

```diff
@@ -11,24 +17,41 @@ public interface IObservableWatchedClient
         /// <param name="owner">The owner of the repository</param>
         /// <param name="name">The name of the repository</param>
         /// <exception cref="AuthorizationException">Thrown if the client is not authenticated</exception>
-        /// <returns>A <see cref="IObservable{User}"/> of <see cref="User"/>s watching the passed repository</returns>
```

Comment:
@Haacked We've discussed this [here](https://github.com/octokit/octokit.net/pull/1344/files/ddb34265bbab715c5e151f4a8e51d05095bc8987#diff-cda94b5dbadd28f0d27ebaff90d53faaL33) with @shiftkey. And yep. I'll remove `/// <returns></returns>` in next commits when other things will be reviwed :+1: